webassembly/example.html

```diff
@@ -1,51 +1,104 @@
 <!DOCTYPE html>
-<html>
+<html data-theme="dark">
 
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>F3D Web</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.0/css/bulma.min.css">
   <style>
-    html,
-    body {
-      height: 100%;
-      margin: 0;
-    }
-
     #main {
-      min-height: 60%;
+      min-height: 70vh;
+      border: 1px solid var(--bulma-primary);
+      border-radius: 0.5rem;
+      overflow: hidden;
     }
   </style>
 </head>
 
 <body>
-  <h1>F3D Web demo</h1>
-  <div id="main">
-    <canvas id="canvas"></canvas>
-  </div>
+  <section class="section">
+    <div class="container">
+      <div class="columns">
+        <div class="column">
+            <h1 class="title">F3D</h1>
+            <p class="subtitle">
+              WebAssembly demo
+            </p>
+        </div>
+
+        <div class="column">
+          <div class="file has-name is-right">
+            <label class="file-label">
+              <input class="file-input" type="file" id="file-selector" />
+              <span class="file-cta">
+                <span class="file-label">Open a 3D file...</span>
+              </span>
+              <span class="file-name" id="file-name">f3d.vtp</span>
+            </label>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="section">
+    <div class="container" id="main">
+      <canvas id="canvas"></canvas>
+    </div>
+  </section>
   <script type="text/javascript" src="f3d.js"></script>
   <script type="text/javascript">
-    var Module = {
-      canvas: (() => {
-        return document.getElementById("canvas");
-      })(),
-      onRuntimeInitialized: () => { },
-    };
+    const settings = { canvas: document.getElementById("canvas") };
```

Comment:
if you're going for a `settings` object to pass the canvas would it make sense to also add the initial options and filename in there?